### DISCUSSION TITLE
Potential dead code

### DISCUSSION THREAD

**Discussion by NMinhNguyen** [react-hook-form/react-hook-form#8336] [General] [answered]
Was just going through the code and spotted a couple of things:

- What’s the point in `Object.keys(_formState.errors).every()` when there’s a check for `_formState.errors` being empty? https://github.com/react-hook-form/react-hook-form/blob/248c7c7187accf2fa040d0f4287ff4915cf66386/src/logic/createFormControl.ts#L1041-L1042
- This module seems to be unused: https://github.com/react-hook-form/react-hook-form/blob/248c7c7187accf2fa040d0f4287ff4915cf66386/src/logic/setCustomValidty.ts

**Comment by bluebill1049** [comment, marked as answer]
good find! please send PR 👍

**Reply by NMinhNguyen** [reply]
Thanks! You definitely didn’t mean `||` instead of `&&` for example?

**Reply by bluebill1049** [reply]
pass test without the `.every` check 🌷
